charts/longhorn-1.1/longhorn/1.1.200+up1.1.2/Chart.yaml

```diff
@@ -3,6 +3,7 @@ annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Longhorn
   catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/rancher-version: <2.5.99-0
```

Comment:
This is a weird one as longhorn supports both 1.1.x and 1.2.x in 2.6, but I think its fine as `longhorn-100.0.0+up1.1.2.tgz` exists in 2.6. We'd want them to move to the 100's and not freshly install this. Same for chart below.